packages/aws-cdk-lib/aws-eks/lib/helm-chart.ts

```diff
@@ -73,6 +74,13 @@ export interface HelmChartOptions {
    */
   readonly timeout?: Duration;
 
+  /**
+   * Whether or not Helm should treat this operation as atomic; if set, upgrade process rolls back changes
+   * made in case of failed upgrade. The --wait flag will be set automatically if --atomic is used.
+   * @default false
+   */
+  readonly atomic?: boolean;
```

Comment:
Let us also update the prop in [README.md](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-eks/README.md#helm-charts) helm-charts section

Reply:
don't think its needed since there is no dedicated section in Readme that points to what all flags are supported in helm, its generally described as
`Helm charts will be installed and updated using helm upgrade --install, where a few parameters are being passed down (such as repo, values, version, namespace, wait, timeout, etc).` which covers this case